src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -381,13 +381,23 @@ const post911Periods = createSelector(
 
 export const servedAfter911 = formData => !!post911Periods(formData).length;
 
-export const isDisabilityPtsd = disability =>
-  lowerCase(disability).includes(PTSD);
+export const isDisabilityPtsd = disability => {
+  if (!disability || typeof disability !== 'string') {
+    return false;
+  }
+
+  const loweredDisability = disability.toLowerCase();
+  return PTSD_MATCHES.some(
+    ptsdString =>
+      ptsdString.includes(loweredDisability) ||
+      loweredDisability.includes(ptsdString),
```

Comment:
I made [a little JS Bin](https://jsbin.com/jobiciyoni/edit?js,console) to test out a theory. Not sure if you want to use it, but I think we can make this use regular expressions as well to accomplish the same task.

Reply:
This is nice; when I wrote this thing I had a suspicion there was a way to do this with regex 😄 
After thinking about it for a bit, I think I'm going to stick with the current approach, for no other reason than the fact that it's subjectively easier for me to keep track of what's going on 😛 .

Reply:
Hehe, 's fine by me. :stuck_out_tongue: